docs/user/getting-started.md

```diff
@@ -47,14 +47,37 @@ NAME  NAMESPACE  REVISION  UPDATED  STATUS  CHART  APP VERSION
 
 ## Deploying the Flight Control Service
 
-### Standalone flightctl with built-in Keycloak
+### Standalone Flight Control on k8s/KIND
+
+Start your k8s/KIND cluster. For KIND cluster you can use [example config](../../deploy/kind.yaml).
+
+```console
+$ kind create cluster --config kind.yaml
+
+[...]
+```
+
+Install a released version of the Flight Control Service into the cluster by running:
+
+```console
+$ helm upgrade --install --version=<version-to-install> \
+    --namespace flightctl --create-namespace \
+    flightctl oci://quay.io/flightctl/charts/flightctl \
+    --set global.baseDomain=127.0.0.1
```

Comment:
Should we use `${YOUR_IP}.nip.io`, so the deployment is reachable from other nodes?

Reply:
sounds good. Updated